docs/customization/form.rst

```diff
@@ -123,6 +123,86 @@ Need more information?
 
     Some of the forms already have extensions in Sylius. Learn more about Extensions `here <http://symfony.com/doc/current/bundles/extension.html>`_.
 
+F.i. the ``ProductVariant`` admin form is defined under ``Sylius/Bundle/ProductBundle/Form/Type/ProductVariantType.php`` and later extended in
+``Sylius/Bundle/CoreBundle/Form/Extension/ProductVariantTypeExtension.php``. If you again extend the base type form like this:
+
+.. code-block:: yaml
+
+    services:
+        app.form.extension.type.product_variant:
+            class: AppBundle\Form\Extension\ProductVariantTypeMyExtension
+            tags:
+                - { name: form.type_extension, extended_type: Sylius\Bundle\ProductBundle\Form\Type\ProductVariantType, priority: -5 }
+
+your form extension will also be executed. Whether before or after the other extensions depends on priority tag set.
+
+Having a look at the extensions and possible additionally defined event handlers can also be useful when form elements are embedded dynamically,
+as it is done in the ``ProductVariantTypeExtension`` by the ``CoreBundle``:
```

Comment:
@CoderMaggie because I believe removing "it" is grammatically correct/provides better flow. I might be wrong though, a native English speaker would have to confirm.